### ISSUE TITLE
Duplicates in cve collection?

### ISSUE THREAD

**Issue by cgkwork** [ALFA-group/BRON#33]
I am puzzled as to occurrence of duplicates in the cve collection.  This is on the demo at http://bron.alfa.csail.mit.edu/.  For example, the following query returns five elements:

FOR c in cve
  FILTER c.original_id == "CVE-2015-9251"
    RETURN c

The only difference in the five elements returned are the _key, _id and _rev.  However, these do not appear to be meaningful differences to me.  Despite searching this repository, I am not able to find an explanation.  Any ideas?

**Comment by hembergerik** [comment]
Yes, it could come from when the database is updated (every 24h). We are trying to find a fix for it. 
A temporary method for avoiding it is to rebuild the database every time it is updated, e.g. if you set up your own following the docker instructions.
